app/plugins/Awards/config/Migrations/20260817120000_PublishRejectedRecommendationClosureWorkflows.php

```
@@ -0,0 +1,188 @@
+    /**
+     * @return void
+     */
+    public function up(): void
+    {
+        $definitionsTable = TableRegistry::getTableLocator()->get('WorkflowDefinitions');
+        $versionsTable = TableRegistry::getTableLocator()->get('WorkflowVersions');
+        $instancesTable = TableRegistry::getTableLocator()->get('WorkflowInstances');
+        $versionManager = new DefaultWorkflowVersionManager();
+
+        foreach ($this->workflowFiles() as $slug => $jsonFile) {
+            $jsonPath = ROOT . '/config/Seeds/WorkflowDefinitions/' . $jsonFile;
```

PHPStan (2.2.7)

[error] Constant ROOT not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)